api/elastic.py

```diff
@@ -386,6 +396,10 @@ def createFilterClausesAndAggregations(self, filterParams):
             formatAggregation = A(
                 'filter', **{'terms': {'editions.formats': formats}}
             )
+        
+        #if len(govDocFilters) > 0:
+
+        #if len(noGovDocFilters) > 0:
```

Comment:
There is a field in the ES index `is_government_document` but at present it does not appear to be populating correctly. I think this _would_ be the easiest way to implement the feature if we can fix it!